### DISCUSSION TITLE
How do I change the CSS file?

### DISCUSSION THREAD

**Discussion by ghost** [cotes2020/jekyll-theme-chirpy#1212] [Q&A] [answered]
I want to change the font-family but every changes I made inside the `style.css` located at `_site/assets/css` gets removed whenever I run `bundle exec jekyll s`

What am I doing wrong? I want to add a google web font called Inter.

**Comment by ghost** [comment]
Also, how do I remove/change the license-wrapper at the end of each post? `This post is licensed under CC BY 4.0 by the author.`

**Reply by ghost** [reply]
Found it inside `_data/locales/en.yml`

**Comment by kungfux** [comment, marked as answer]
> I want to change the font-family but every changes I made inside the `style.css` located at `_site/assets/css` gets removed whenever I run `bundle exec jekyll s`
>
> What am I doing wrong? I want to add a google web font called Inter.

Well, first of all you should not make any changes inside `_site` directory. As you might noticed, this folder is not a part of this repository. It's being created when you build your instance of the site, so it is fully generated from the source code. Taking this into account, you have to make changes in the source code instead of generated files.

Making changes in the source code files depends on what exact thing you would like to change as things resides in different places. Also, keep in mind that this is not a blog engine rather than blog theme. This means that some things are created specifically to look like that were designed. However, there are ways to change things except modifying `_config.yml`. Besides configuration file, there are mechanisms in place that allows you to make customizations like `assets/css/jekyll-theme-chirpy.scss` allowing to add custom styles.

Typically, when you want to override something, you need to copy a file from original repo and place it at the same place in your fork and modify as you wish. However, doing so will mean that you are the only person responsible to support these changes and make sure they still work on every blog version update.

In this particular case, please refer to the [answer](https://github.com/cotes2020/jekyll-theme-chirpy/discussions/1185#discussioncomment-6775249) given already.

Hope this helps someone.
